tests/check/signed_video_helpers.c

```diff
@@ -57,6 +57,8 @@ const char *axisDummyCertificateChain =
     "w3S8HMYSvMWbTCzN+qnq+GV1goSS6vjVr95EpDxCVIxkKOvuxhyVDg==\n"
     "-----END CERTIFICATE-----";
 
+const int64_t TIMESTAMP = 42;
```

Comment:
Capital letters are for defines. At one point it was discussed to prepend names of constants with a `k`, but I see it is not fully compliant. Go for either `timestamp`, `kTimestamp` or something like `testTimestamp`.

Reply:
C convention seems to like prefixing with "g_". Changing to g_testTimestamp

Reply:
Convention in C seems to be g_camelCase, changed to g_testTimestamp